src/IotHub/IotHub/IotHub/SetAzureRmIotHub.cs

```diff
@@ -121,6 +122,8 @@ public class SetAzureRmIotHub : IotHubBaseCmdlet
         [ValidateNotNullOrEmpty]
         public PSCloudToDeviceProperties CloudToDevice { get; set; }
 
+        public const string ChangeDesc = "Parameter is being deprecated without being replaced.";
```

Comment:
Same comment.  This is not a good enough user description.

Reply:
Updated the description